docs/content/configuration/index.md

```diff
@@ -379,6 +379,7 @@ For native query, only request logs where query/time is above the threshold are
 |--------|-----------|-------|
 |`druid.request.logging.queryTimeThresholdMs`|Threshold value for query/time in milliseconds.|0 i.e no filtering|
 |`druid.request.logging.sqlQueryTimeThresholdMs`|Threshold value for sqlQuery/time in milliseconds.|0 i.e no filtering|
+|`druid.request.logging.queryTypeBlacklist` | List of Druid query types to exclude from request logs. Query types are defined as string objects in the Query interface found in Query.java. Mispelled query types will be ignored. Example to ignore scan and timeBoundary queries: ["scan", "timeBoundary"]| []|
```

Comment:
`queryTypeBlacklist` sounds a little intimidating. I would say "mutedQueryTypes".

Reply:
I suggest the description:
```
```suggestion
|`druid.request.logging.mutedQueryTypes` | Query requests of these types are not logged. Query types are defined as string objects corresponding to the "queryType" value for the specified query in the Druid's [native JSON query API](http://druid.apache.org/docs/latest/querying/querying.html). Misspelled query types will be ignored. Example to ignore scan and timeBoundary queries: ["scan", "timeBoundary"]| []|
```